controls/radgauge/getting-started/creating-linear-gauge.md

```diff
@@ -56,17 +46,21 @@ Here is an example of a single __LinearScale__:
 	</telerik:RadVerticalLinearGauge>
 {{endregion}}
 
-Here is a snapshot of the result:
+>tip To learn more about the usage of the __LinearScale__ control, read the [Linear Scale]({%slug linear-scale-overview%}) topic.
+
+#### __Image 1: Result from Example 1__
 
-![](images/RadGauge_GettingStarted_LinearGauge_02.png)
+![RadVerticalLinearGauge with a VerticalLinearScale](images/RadGauge_GettingStarted_LinearGauge_02.png)
+
+> The examples in this article are styled with the [Windows8]({%slug common-styles-appearance-colorizing-metro-theme%}) theme.
 
 ## Defining an Indicator
 
-The __LinearScale__ control allows you to display different types of indicators that point to a value off the scale. To specify an indicator use the __Indicators__ property, which is of type __ItemCollection__. This means that the __Indicators__ property can hold more than one indicator. Here is an example of LinearScale that uses a Marker indicator to mark the current value.
+The __LinearScale__ control allows you to display different types of indicators that point to a value on the scale. To specify an indicator use the __Indicators__ property, which is of type __ItemCollection__. This means that the __Indicators__ property can hold more than one indicator. Here is an example of LinearScale that uses a Marker indicator to mark the current value.

```

Comment:
To specify an indicator, use...
... __Example 2__ shows how you can define a LinearScale that uses.....